lib/travis/build/script/julia.rb

```diff
@@ -58,31 +58,38 @@ def setup
         def announce
           super
 
-          sh.cmd "julia -e 'versioninfo()'"
+          sh.cmd 'julia --color=yes -e "VERSION >= v\"0.7.0-DEV.3630\" && using InteractiveUtils; versioninfo()"'
           sh.echo ''
         end
 
         def script
           sh.echo 'Executing the default test script', ansi: :green
-          set_jl_pkg
+
+          # Extract the package name from the repository slug (org/pkgname.jl)
+          m = /(\w+?)\/(\w+?)(?:\.jl)?$/.match(ENV['TRAVIS_REPO_SLUG'])
```

Comment:
Please let me know if there is a better way to get the repo name.

Reply:
The environment variable `TRAVIS_REPO_SLUG` exists in the shell, yet here it is used in the ruby context. I think you are after `data[:repository][:slug]` here.

Reply:
Ah, that makes sense. Can you stage again please?